packages/builder/src/builderStore/dataBinding.js

```diff
@@ -391,6 +397,16 @@ const getUrlBindings = asset => {
   }))
 }
 
+const getRoleBindings = () => {
```

Comment:
awesome this was straight forward. just to check, do we get the custom roles from the `roleStore`?

Reply:
I think we removed the idea of custom roles... At least I couldn't find them in the UI when I went to check it haha. Either way I do think they are in this role store yea.

Reply:
I ran into this the other day, it wasn't so clear but they are there. You need to add the role in-app to the users table using `Edit Roles`. That role can then be assigned on the edit user page in the portal when assigning a role for that app with the custom role.

Reply:
Could be worth a quick check just to see how it plays, as custom roles were the rationale for the original copy/paste role id issue. We can probably just close this pr https://github.com/Budibase/budibase/pull/3468 when we merge this new change instead

Reply:
Haha you'd think I'd know where to find it.

Yep works with custom roles:
![image](https://user-images.githubusercontent.com/9075550/161952849-717bcde3-98a6-499d-8bfe-af56f3daa204.png)

Reply:
And yea agree about closing that PR once this is in.

Reply:
nice! this is great